holo-vrrp/src/northbound/configuration.rs

```diff
@@ -234,6 +238,9 @@ impl Provider for Interface {
                     );
                     instance.timer_set(&interface);
                 }
+                // owner status may change when virtual address is added
```

Comment:
Owner status can also change when an IP address is added to or removed from the interface.

For simplicity, we could compute the owner status on demand whenever necessary, instead of caching and continuously updating it. There's only a few places where we need to consult the owner status, and computing it is a cheap operation.

Reply:
@rwestphal You're right, the owner status can also changed when an IP address is added or removed from an interface. 

Looking through I added [this](https://github.com/holo-routing/holo/blob/db002182beda64434f1a9d952f8634ac3eb34758/holo-vrrp/src/instance.rs#L169C1-L169C14) in the `startup()` method when it should be in the `update()` method. `instance.update()` is the only method that is called when there is a change in either the parent interface or the instance's Macvlan interface.